.github/workflows/ci.yaml

```diff
@@ -18,14 +18,29 @@ jobs:
         working-directory: ${{ env.WORKING_DIR }}
     strategy:
       matrix:
-        go: ['1.14', '1.15']
```

Comment:
Mention eliminating 1.14 in description?

Reply:
Updated the PR title.